public/data/CyberDevv.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "Odesola Ibrahim",
+  "type": "personal",
+  "bio": "I am a Full-Stack Developer using React and Node.js",
+  "avatar": "https://github.com/cyberdevv.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/cyberdevv",
+      "icon": "github"
+    },
+
+    {
+      "name": "Connect with me on Linkedin",
+      "url": "https://www.linkedin.com/in/odesolaibrahim/",
+      "icon": "linkedin"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/cyberdevv",
+      "icon": "twitter"
+    },
+    {
+      "name": "Checkout my portfolio",
+      "url": "https://ibrahim-odesola.netlify.app/",
```

Comment:
```suggestion
      "url": "https://ibrahim-odesola.netlify.app",
```

Reply:
This doesn't matter at all, IMO. It can be kept. 🙂